src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -3852,7 +3852,13 @@ In certain complex graphs or host program scenarios, Automatic mode may cause in
   <data name="ElementBindingDesc" xml:space="preserve">
     <value>Element Binding allows a two-way interaction between Dynamo and the host application like Revit or Civil3D where a user can select an element in the host document and have Dynamo "bind" that element to a node in the workspace.</value>
   </data>
+  <data name="PreferencesViewVisualSettingUseInstancing" xml:space="preserve">
+    <value>Use Instancing (Beta)</value>
+  </data>
+  <data name="PreferencesWindowUseInstancingTooltip" xml:space="preserve">
+    <value>When toggled on, some graphics will render faster</value>
```

Comment:
I feel we can explain that it's due to instancing, and what happens during instancing?